boms/cloud-oss-bom/RELEASING.md

```diff
@@ -32,6 +32,7 @@ version in these documents:
 * https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM
 * https://github.com/googleapis/google-http-java-client/blob/master/docs/setup.md
 * https://github.com/googleapis/google-cloud-java/blob/master/TROUBLESHOOTING.md
+* All pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples
```

Comment:
Do you want to provide some command (`find` or `grep`?) here so that executors (you and me in mind) perform the same operations?

Reply:
Do we have to wait for auto-update performed before manually editing the page?

Reply:
Possibly.